scripts/asm-installer/install_asm

```diff
@@ -450,7 +456,11 @@ OPTIONS:
                                       omitted if not installing optional
                                       features. To add multiple files, specify
                                       them with multiple options one at a time.
-
+  --ca_id|--ca-id         <CA_ID>     Required only if --ca option is private_ca.
```

Comment:
a note on terminology - the resource *ID* is just the last part of the full string, while the resource *name* is the `projects/foo/locations/bar/certificateAuthorities/baz` string you described below. In this case, I think they are flipped around.

more context: https://google.aip.dev/122

Reply:
Thanks for the clarification! Will rename it to resource name